### PR TITLE
tiflash: add a new configuration dt_enable_read_thread

### PR DESCRIPTION
**Pull request by JinheLin**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add TiFlash configuration.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -182,6 +182,8 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 使用线程池处理存储引擎的读请求。默认为 false。
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:185]:
```suggestion
    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -182,6 +182,8 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:185]:
```suggestion
    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。**警告：目前是实验性功能，不建议在生产环境中使用。**
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -182,6 +182,8 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。**警告：目前是实验性功能，不建议在生产环境中使用。**
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:185]:
```suggestion
    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。警告：目前是实验性功能，不建议在生产环境中使用。
```

**Comment by JinheLin** [comment]
@flowbehappy PTAL

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -182,6 +182,8 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。警告：目前是实验性功能，不建议在生产环境中使用。
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:185]:
```suggestion
    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。
    ## 警告：目前是实验性功能，不建议在生产环境中使用。
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -182,6 +182,9 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v6.2 引入，使用线程池处理存储引擎的读请求。默认为 false。
```
Comment (ran-huang) [inline tiflash/tiflash-configuration.md:185]:
```suggestion
    ## 从 v6.2.0 引入，使用线程池处理存储引擎的读请求。默认为 false。
```

**Pull request merged**